src/libraries/System.ObjectModel/tests/TrimmingTests/TypeDescriptionProviderAttributeCtor.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.ComponentModel;
+using System.Xml.Linq;
+
+class Program
+{
+    static int Main(string[] args)
+    {
+        // The public parameterless ctor of the Type argument here is preserved.
+        TypeDescriptionProviderAttribute attr = new TypeDescriptionProviderAttribute(typeof(MyTypeDescriptionProvider));
```

Comment:
We should test both constructors - the one that takes a Type and the one that takes a string.

Reply:
Note: to correctly test both each will have to use a different type (otherwise success of one would automatically make the other work as well, without actually testing it).